tests/test_jsonwrite.py

```diff
@@ -0,0 +1,60 @@
+import temporalimage
+import unittest
+import jsonschema
+from jsonschema import validate
+import numpy as np
+from temporalimage import Quantity
+from temporalimage.t4d import _jsonread_frameTiming
+from temporalimage.t4d import _jsonwrite_frameTiming
+
+
+#The following code test the _jsonwrite_frameTiming() function to see whether t4d.py 
+#has correctly converted other json formats into our json schema.
+
+
+#We use jasonschema library. 
+#Creat a json schema following our json format
+#Validate the json document against the schema with our format
+#if it passes, we could say that the json document is correctly converted to our format from whichever versions we are given.
+#install jsonschema using pip command
+#pip install jsonschema
+
+#Define Schema-we expect the following json schema:
+#FrameTimesStart, FrameTimesStartUnits, FrameDuration and FrameDurationUnits must be present in json data.
+ourSchema = {
+    "properties":{
+        "FrameTimesStart": {"type": "array"},
+        "FrameTimesStartUnits":{"type": "string"},
+        "FrameDuration":{"type": "array"},
+        "FrameDurationUnits":{"type": "string"}
+    },
+}
+
+import simplejson as json
+from json import load as json_load	
+jsonTestFile = "/opt/anaconda3/envs/toydataset/pet-analysis/tests/test_data/ds001420/sub-01/ses-baseline/pet/sub-01_ses-baseline_pet.json"
```

Comment:
This path is hard coded; it will not run for other users. You need to find a way to make this path valid for other users.